### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR:
 - adds configuration needed for deployment
 - defaults DEBUG to False

### CODE REVIEW

File: back/.gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Directories
-pyvenv/
+.venv/
```
Comment (mparkan) [inline back/.gitignore]: Currently using "venv/", not ".venv/"

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Dependency directories
 .vite/
 back/pyvenv/
-back/.vscode
\ No newline at end of file
+back/.venv/
```
Comment (mparkan) [inline .gitignore]: Currently using "venv/", not ".venv/"

**Review by mparkan — CHANGES_REQUESTED** [review]
Please change ".venv/" to "venv/" :o)

**Review by mparkan — APPROVED** [review]

**Comment by maltaesousa** [comment]
> Please change ".venv/" to "venv/" :o)

Yes, you're right , I've also updated wsgi.py, if ok for you, I merge

**Review by mparkan — APPROVED** [review]
LGTM, merci :o)

**Pull request merged**